_config/documentconverter.yml

```diff
@@ -0,0 +1,14 @@
+---
+Name: cwpdocumentconverterconfig
+Only:
+  moduleexists: silverstripe/documentconverter
+  EnvVarSet: DOCVERT_USERNAME
+---
+Silverstripe\DocumentConverter\DocVertConnector:
```

Comment:
During our peer session we changed the classes to drop the Document/docvert/conversion prefixes
So this has been updated to `SilverStripe\DocumentConverter\ServiceConnector`
The rest is all good though :)

Reply:
Updated in 29d8be2